src/system_info/system_info_instance.cc

```diff
@@ -38,7 +38,13 @@
 #include "system_info/system_info_device_orientation.h"
 #include "system_info/system_info_display.h"
 #include "system_info/system_info_locale.h"
-#include "system_info/system_info_network.h"
+#if defined(GENERIC_DESKTOP)
+#include "system_info/system_info_network_desktop.h"
+#elif defined(TIZEN_MOBILE)
+#include "system_info/system_info_network_mobile.h"
+#else
+#include "system_info/system_info_network_ivi.h"
+#endif
```

Comment:
Good progress, this ifdef also can be avoided. Please enhance `src/common/comm.gypi` to include/exclue the header files.